src/client/app/components/about.ts

```diff
@@ -41,45 +41,45 @@ export class About implements AfterViewInit {
     @Input() show: boolean;
     @Output() showChange = new EventEmitter<boolean>();
 
+    cache = `
+    ${Strings.aboutStorage}
+    ${storageSize(localStorage, `playground_${environment.current.host}_snippets`, Strings.aboutSnippets)}
+    ${storageSize(sessionStorage, 'playground_intellisense', Strings.aboutIntellisense)}
+    `;
+
+    config = {
+        build: environment.current.build,
+        editorUrl: environment.current.config.editorUrl,
+    };
+
+    configs = [];
+
+    selectedConfig = '';
+
     constructor(
         private _effects: UIEffects,
         private _changeDetector: ChangeDetectorRef
     ) { }
 
     ngAfterViewInit() {
         if (environment.current.config.name === config['local'].name) {
-            this.configs.push({ name: 'local', value: 'Localhost' });
+            this.configs.push({ name: 'local', value: config['local'].editorUrl });
         }
         this.configs.push(
             { name: 'production', value: 'Production' },
-            { name: 'edge', value: 'Beta' },
-            { name: 'insiders', value: 'Alpha' },
+            { name: 'edge', value: 'Alpha' },
```

Comment:
Could you order these in the order of **Production -> Beta -> Alpha** in the dropdown?  Also, move the "if" statement for localhost down after this block here, so that for localhost it goes **Production -> Beta -> Alpha -> https://localhost:3000**